style.css

```
@@ -0,0 +1,83 @@
+*,
+*::before,
+*::after {
+  margin: 0;
+  padding: 0;
+  box-sizing: inherit;
+}
+
+html {
+  box-sizing: border-box;
+  font-size: 62.5%;
```

It's always good to leave a comment on why default things get changed, like in this case a comment like this would be great info for the team
```suggestion
  // Setting the page base font to 10px to simplify `rem` calculations by making them a multiple of 10
  font-size: 62.5%;
```